code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -417,7 +418,10 @@
 	icon_state = "bamboo_sofamiddle"
 	color = null
 	colorable = FALSE
-
+	resistance_flags = FLAMMABLE
+	max_integrity = 70
+	buildstackamount = 2
+	buildstacktype = /obj/item/stack/sheet/wood
 /obj/structure/chair/sofa/bamboo/left
```

Comment:
```suggestion
	buildstacktype = /obj/item/stack/sheet/wood
	
/obj/structure/chair/sofa/bamboo/left
```
spacing

Reply:
yeah, just saw that missing gap, fixing those now